internal/uri/uri_util_test.go

```diff
@@ -17,6 +17,7 @@
 package uri
 
 import (
+	"github.com/snyk/snyk-ls/internal/testutil"
```

Comment:
This import should be grouped separately below (first-party imports). I think there should be a linter rule that checks for this (gci?)

Reply:
I would refrain from using testutil at all in internal packages to avoid circular dependencies.